pillar/flavors/production.sls

```diff
@@ -43,7 +43,8 @@
             "taskmanager_heapsize": 2048,
             "taskmanager_slots": 1,
             "parallelism": 1,
-            "taskmanager_mem_preallocate": false
+            "taskmanager_mem_preallocate": false,
+            "pyflink_yarn_container_count": 1
```

Comment:
Would there be any reason to set this higher than 1 for standard / production flavours? If so, then do it here, if not then why does it need to be a flavour specific parameter?

Reply:
Like other parameters namely heap-size, parallelism and available slots, instead of hard-coding we have kept it configurable, so that it can be configured or fine tuned appropriately. 
I think going ahead considering type of applications we are running, we will fine tune the value so that by default it will be used for maximum data-heavy applications - without any explicit user configurations.